src/bam/record.rs

```diff
@@ -306,6 +306,19 @@ impl Record {
         }
     }
 
+    // Delete auxiliary tag. Return false if tag doesn't exist
+    pub fn del_aux(&self, tag: &[u8]) -> bool {
```

Comment:
You are right, it is not an error. Thinking about it, it is more like the [retain method](https://doc.rust-lang.org/beta/std/vec/struct.Vec.html#method.retain). We remove stuff, but no value is returned. Rust does not provide a return type there. Hence, I would vote for doing the same.